src/components/TextField/TextField.test.tsx

```diff
@@ -49,43 +49,62 @@ describe('<TextField />', () => {
       expect(component).toMatchSnapshot();
     });
   });
-});
 
-describe('TextField#focus', () => {
-  let ref: any;
-  const setRef = (i: any) => (ref = i);
+  describe('full mount with default options', () => {
+    let ref: any;
+    const setRef = (i: any) => (ref = i);
 
-  describe('with default options', () => {
-    beforeEach(() => {
-      mount(<TextField componentRef={setRef} />);
-    });
+    beforeEach(() => {});
+
+    describe('when focused', () => {
+      let focusSpy: jest.SpyInstance;
+      let setSelectionRangeSpy: jest.SpyInstance;
 
-    it('focus', () => {
-      const focusSpy = jest.spyOn(FabricTextField.prototype, 'focus');
-      const setSelectionRangeSpy = jest.spyOn(FabricTextField.prototype, 'setSelectionRange');
+      beforeEach(() => {
+        focusSpy = jest.spyOn(FabricTextField.prototype, 'focus');
+        setSelectionRangeSpy = jest.spyOn(FabricTextField.prototype, 'setSelectionRange');
+        mount(<TextField componentRef={setRef} />);
+      });
 
-      ref.focus();
+      it('calls focus', () => {
+        ref.focus();
```

Comment:
this action should also be up in the beforeEach. otherwise the describe block 'when focused' is sort of a lie. and you wouldn't have to repeat the focus action for each expectation

Reply:
I agree. Thanks for catching this.